ffmap/routertools.py

```diff
@@ -17,7 +17,8 @@
 
 CONFIG = {
 	"vpn_netif": "fffVPN",
-	"vpn_netif_l2tp": "l2tp2",
+	"vpn_netif_l2tp": "l2tp0",
```

Comment:
* Inconsistent naming
* I think we should use a more generic approach, instead of adding support for specific _l2tp**X**_ devices